src/Resources/config/actions.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\AdminBundle\Action\AppendFormFieldElementAction;
+use Sonata\AdminBundle\Action\DashboardAction;
+use Sonata\AdminBundle\Action\GetShortObjectDescriptionAction;
+use Sonata\AdminBundle\Action\RetrieveAutocompleteItemsAction;
+use Sonata\AdminBundle\Action\RetrieveFormFieldElementAction;
+use Sonata\AdminBundle\Action\SearchAction;
+use Sonata\AdminBundle\Action\SetObjectFieldValueAction;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\DependencyInjection\Reference;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->services()
+
+        ->set(DashboardAction::class, DashboardAction::class)
+            ->public()
+            ->args([
+                '%sonata.admin.configuration.dashboard_blocks%',
```

Comment:
you should use https://github.com/symfony/symfony/blob/6288449f612e5a08f7ae0ea40bac9d20454b88e6/src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php#L89

Reply:
this is also related to your problem @core23 in some way

Reply:
`param` function [will be available on Symfony 5.2](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/CHANGELOG.md)